### PR TITLE
Add supported tokens

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-commons/lib/types/common.ts [hunk 1]
```diff
@@ -26,3 +28,14 @@ export declare interface ContractWhiteList {
   master: string[];
   proxy: string[];
 }
+
+export interface SupportedToken {
+  address: string;
+  minimalAmount: utils.BigNumber;
+}
+
+export type ChainSpec = {
```
Comment (marekkirejczyk) [inline universal-login-commons/lib/types/common.ts]: Move to Config

File: universal-login-relayer/lib/relayer.ts [hunk 2]
```diff
@@ -69,9 +69,15 @@ class Relayer {
     this.pendingMessagesStore = new PendingMessagesSQLStore(this.database);
     this.messageQueueStore = new MessageQueueStore(this.database);
     this.messageHandler = new MessageHandler(this.wallet, this.authorisationService, this.hooks, this.pendingMessagesStore, this.messageQueueStore, this.config.contractWhiteList);
+    const {chainSpec, supportedTokens, factoryAddress} = this.config;
+    const config = {
+      chainSpec,
+      supportedTokens,
+      factoryAddress
+    };
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/relayer.ts]: extract getPublicConfig(Config config) : PublicConfig

File: universal-login-relayer/test/relayer/routes/config.js [hunk 3]
```diff
@@ -12,10 +12,15 @@ describe('E2E: Relayer - Config routes', async () => {
   });
 
   it('Config', async () => {
-    const expectedEnsAddress = relayer.config.chainSpec.ensAddress;
+    const {supportedTokens, chainSpec, factoryAddress} = relayer.config;
+    const expectedConfig = {
+      supportedTokens,
+      chainSpec,
+      factoryAddress
+    };
```
Comment (marekkirejczyk) [inline universal-login-relayer/test/relayer/routes/config.js]: Use `getPublicConfig`

File: universal-login-commons/lib/types/common.ts (hunk 1 above)
Comment (xitronix) [reply universal-login-commons/lib/types/common.ts]: ✔️

File: universal-login-relayer/test/relayer/routes/config.js [hunk 4]
```diff
@@ -12,10 +13,27 @@ describe('E2E: Relayer - Config routes', async () => {
   });
 
   it('Config', async () => {
-    const expectedEnsAddress = relayer.config.chainSpec.ensAddress;
+    const {supportedTokens, chainSpec, factoryAddress} = relayer.config;
```
Comment (marekkirejczyk) [inline universal-login-relayer/test/relayer/routes/config.js]: Use `getPublicConfig`

**Pull request merged**